### ISSUE TITLE
Heroes tutorial toh-pt2 mentions a let statement, but uses # in the template

### ISSUE THREAD

**Issue by sturmf** [angular/angular.io#1648]
In the tutorial it mentions this:

> The let keyword before "hero" identifies the hero as a template input variable. We can reference this variable within the template to access a hero’s properties.

but uses this in the code example:

`<li *ngFor="#hero of heroes">`

so I guess #variable equals let variable ?? I think this should be mentioned or only use the let syntax in the example.

**Comment by Foxandxss** [comment]
No, that is wrong. I will fix it.

**Comment by sturmf** [comment]
It's in the example that you are supposed to copy paste:
```

app_component.dart (Capturing the click event)

<li *ngFor="#hero of heroes" (click)="onSelect(hero)">
  <span class="badge">{{hero.id}}</span> {{hero.name}}
</li>
```

There you see the usage of the # but the text talks about a let statement.
From: https://angular.io/docs/dart/latest/tutorial/toh-pt2.html

**Comment by Foxandxss** [comment]
Ohhh, dart. My bad. Ping @chalin

**Comment by chalin** [comment]
It is because of the snippets :-/. I'll fit it soon.

**Comment by chalin** [comment]
@Foxandxss : fixed by #1941.
